src/sentry/api/endpoints/organization_events_trendsv2.py

```diff
@@ -194,14 +197,40 @@ def get_trends_data(stats_data, request):
 
             trends_request["sort"] = request.GET.get("sort", "trend_percentage()")
             trends_request["trendFunction"] = trend_function
-            trends_request["data"] = stats_data
+
+            # list of requests to send to microservice async
+            trends_requests = []
+
+            # split the txns data into multiple dictionaries
+            split_transactions_data = [
+                dict(list(stats_data.items())[i : i + events_per_query])
+                for i in range(0, len(stats_data), events_per_query)
+            ]
+
+            for i in range(len(split_transactions_data)):
+                trends_request["data"] = split_transactions_data[i]
```

Comment:
`trends_request` needs to be copied before it's mutated and appended to `trends_requests`. otherwise, you're overwriting the original `trends_request["data"]` 
```suggestion
                trends_request = trends_request.copy()
                trends_request["data"] = split_transactions_data[i]
```

Reply:
so if `split_transactions_data` length is 3, the final value of `trends_request["data"]` is included three times instead of 3 different values